### DISCUSSION TITLE
Add support for PEFT fine-tuning

### DISCUSSION THREAD

**Discussion by alvaropp** [amazon-science/chronos-forecasting#186] [Ideas]
Hi 👋🏽 

I've been fine-tuning Chronos on my own data and modified the code to support fine-tuning with PEFT (LoRA) for efficiency. Is there an interest to try get these changes into the repo? There seem to be some earlier discussions asking about this.

**Comment by lostella** [comment]
@alvaropp that's very interesting, I'd be curious to know how well that works. As for contributing the code however, I think we prefer to keep this repository as related to the paper experiments as possible. Hope this makes sense!

**Reply by alvaropp** [reply]
Yeah, totally. If time allows I may put my PEFT and inference code somewhere, I'll share it here 👍🏽
